README.md

```diff
@@ -14,3 +13,103 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Build a Docker containerized, secure Java Web API application using Managed Identity, Key Vault, and Cosmos DB that is designed to be deployed to Azure App Service or AKS
+
+![License](https://img.shields.io/badge/license-MIT-green.svg)
+![Docker Image Build](https://github.com/retaildevcrews/helium-typescript/workflows/Docker%20Image%20Build/badge.svg)
+
+This is a Java Spring Boot Web API reference application designed to "fork and code" with the following features:
+
+- Securely build, deploy and run an App Service (Web App for Containers) application
+- Use Managed Identity to securely access resources
+- Securely store secrets in Key Vault
+- Securely build and deploy the Docker container from Container Registry
+- Connect to and query CosmosDB
+- Automatically send telemetry and logs to Azure Monitor
+- Instructions for setting up Key Vault, ACR, Azure Monitor and Cosmos DB are in the Helium [readme](https://github.com/retaildevcrews/helium)
+
+## Prerequisites
+
+- Azure subscription with permissions to create:
+  - Resource Groups, Service Principals, Keyvault, CosmosDB, App Service, Azure Container Registry, Azure Monitor
+- Bash shell (tested on Mac, Ubuntu, Windows with WSL2)
+  - Will not work in Cloud Shell unless you have a remote dockerd
+- Azure CLI 2.0.72+ ([download](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- Docker CLI ([download](https://docs.docker.com/install/))
+- Java 8 above ([download](https://nodejs.org/en/download/)) 
```

Comment:
Java is pointing to node download link, needs to point to correct jdk download link.
Also Maven download link needs to be added

Reply:
Good catch thank you - corrected to:
- Java 8 above ([download](https://www.java.com/en/download/manual.jsp/)) 
and added below link:
- Maven ([download](https://maven.apache.org/download.cgi))